### PR TITLE
[author] Update country-region-dropdown-menu

### PR DESCRIPTION
**Pull request by geodatasource**
# Pull request template for new library

Pull request for issue: #
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

## Profile of the lib

* Git repository (required): https://github.com/geodatasource/country-region-dropdown-menu
* Official website (optional, not the repository): http://www.geodatasource.com/
* NPM package url (optional): https://www.npmjs.com/package/country-region-dropdown-menu
* License and its reference: https://github.com/geodatasource/country-region-dropdown-menu/blob/master/license.txt
* GitHub / Bitbucket popularity (required):
  * Count of watchers: 1
  * Count of stars: 15
  * Count of forks: 15
* NPM download stats (optional):
  * Downloads in the last day:
  * Downloads in the last week: 
  * Downloads in the last month:

## Essential checklist

* [x] I'm the author of this library
  * [x] I would like to add link to the page of this library on CDNJS on website / readme
* [ ] This lib was not found on cdnjs repo
* [ ] Doesn't already exist / has duplicate issue and PR
* [ ] The lib has notable popularity
  * [ ] More than 200 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
  * [ ] More than 800 downloads stats per month on npm registry
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)

## Auto-update checklist

* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

## Git commit checklist

* [x] The first line of commit message is less then 50 chars; clean, clear and easy to understand.
* [x] The parent of the commit(s) in the PR is not older than 3 days.
* [x] Pull request is sent from a non-master branch with a meaningful name.
* [x] Separate unrelated changes into different commits.
* [ ] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
* [ ] Close corresponding issue in commit message
* [ ] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by geodatasource** [comment]
Hi @PeterDaveHello 

Could you please help us on this?

Thank you.

File: ajax/libs/country-region-dropdown-menu/package.json [hunk 1]
```diff
@@ -20,14 +20,14 @@
     "target": "git://github.com/geodatasource/country-region-dropdown-menu.git",
     "fileMap": [
       {
-        "basePath": "dist",
+        "basePath": "assets/js",
```
Comment (PeterDaveHello) [inline ajax/libs/country-region-dropdown-menu/package.json]: As there are many files there, maybe we need to add the whole `assets` ?

Comment (geodatasource) [reply ajax/libs/country-region-dropdown-menu/package.json]: Sure, will update about it. Is there any way to add the [language](https://github.com/geodatasource/country-region-dropdown-menu/tree/master/languages) folder too?

**Comment by geodatasource** [comment]
@PeterDaveHello We have updated package.json. 

Thank you.

File: ajax/libs/country-region-dropdown-menu/package.json (hunk 1 above)
Comment (PeterDaveHello) [reply ajax/libs/country-region-dropdown-menu/package.json]: Yes, with another filemap block.

**Comment by geodatasource** [comment]
Hi @PeterDaveHello 

We have removed it. Thank you.

File: ajax/libs/country-region-dropdown-menu/package.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "country-region-dropdown-menu",
   "filename": "geodatasource-cr.min.js",
-  "version": "1.2.2",
+  "version": "2.1.1",
```
Comment (PeterDaveHello) [inline ajax/libs/country-region-dropdown-menu/package.json]: This change needs rollback, thanks.

Comment (geodatasource) [reply ajax/libs/country-region-dropdown-menu/package.json]: Hi @PeterDaveHello 
You mean require us to rollback?

Comment (geodatasource) [reply ajax/libs/country-region-dropdown-menu/package.json]: Hi @PeterDaveHello,

Any update on this?

Comment (MattIPv4) [inline ajax/libs/country-region-dropdown-menu/package.json]:
```suggestion
  "version": "1.2.2",
```
Please revert the change to the version number.

File: ajax/libs/country-region-dropdown-menu/package.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "country-region-dropdown-menu",
   "filename": "geodatasource-cr.min.js",
```
Comment (MattIPv4) [inline ajax/libs/country-region-dropdown-menu/package.json]:
```suggestion
  "filename": "js/geodatasource-cr.min.js",
```

This file appears to now be within the `js` folder (within the `assets` directory which is the basePath for auto-update).

https://github.com/geodatasource/country-region-dropdown-menu/blob/2.1.6/assets/js/geodatasource-crc.min.js
